CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.22.0 Release
+- Release Date: August 16, 2019
+- Server Versions Supported: Server v5.9+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Highlights
+
+#### React Native Navigation
+
+#### Android Q Improvements
```

Comment:
Support for iOS13 and Android Q

Added support for iOS13 and Android Q to be released later this year.